docs/Example Schedules/CE Schedules/compe-kteal2.md

```diff
@@ -0,0 +1,185 @@
+# Computer Engineering, Statistics Minor, 7-8 Semesters
+
+Hi, I'm Keegan, currently a second-semester Junior (CE '25) and I'm pursuing a Statistics minor. I'm focused on software, robotics, autonomy, and controls. Originally, I had planned to use one semester as a co-op but decided to take more technical electives instead. My schedule is definitely on the heavy side but it allows you to spend most of Junior and Senior year taking tech electives that you're interested in, while never having to double up on the big 4 ECE classes.
+
+## Features of My Schedule
+
+- Can be easily modified to 7 semesters (or less)
+- No summer or winter classes, so internship options every summer
+- Statistics minor + James Scholar
+
+## Schedule
+
+*Difficulty rating is subjective to the time I took the courses, so YMMV*
```

Comment:
This seems like it will apply to every example schedule, so I will probably add a similar disclaimer on the index page.  So, you can delete this.